libtransmission/transmission.h

```diff
@@ -1190,6 +1190,7 @@ struct tr_peer_stat
     char addr[TR_INET6_ADDRSTRLEN];
     char flagStr[32];
     char const* client;
+    std::string encoded_peer_id_;
```

Comment:
As mentioned in https://github.com/transmission/transmission/pull/7514#discussion_r2032229339, please keep the peer ID in `tr_peer_stat` raw.